packages/proxy/schema/models.ts

```diff
@@ -1000,11 +1000,27 @@ export const AvailableModels: { [name: string]: ModelSpec } = {
     multimodal: true,
   },
   "gemini-2.0-flash-exp": {
+    format: "google",
+    flavor: "chat",
+    input_cost_per_mil_tokens: 0.1, // TODO: Appears to be free for now?
+    output_cost_per_mil_tokens: 0.4,
+    displayName: "Gemini 2.0 Flash Exp",
```

Comment:
Can we drop the "Exp" from the name - I think it's now GA and no longer experimental